src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/account_filter.py

```diff
@@ -48,7 +48,7 @@ def update_account_filter(instance, start_timestamp=None, end_timestamp=None,
         raise CLIError('The account filter resource was not found.')
 
     if first_quality or bitrate:
-        first_quality = FirstQuality(bitrate=first_quality or bitrate)
+        instance.first_quality = FirstQuality(bitrate=first_quality or bitrate)
```

Comment:
It would be good to add a new scenario test or enhance an existing one to cover this usage.

Reply:
There are tests for these scenarios, but I didn't delete the recordings, so they passed locally. The nightly run caught this.

Reply:
Ah, yes, I see how the recorded test would not catch this.

Reply:
I was pretty embarrassed when I saw the nightly run.